src/display/display_utils.js

```diff
@@ -685,6 +692,23 @@ class OutputScale {
   static get pixelRatio() {
     return globalThis.devicePixelRatio || 1;
   }
+
+  static getCappedWindowArea(capAreaFactor) {
+    if (typeof PDFJSDev !== "undefined" && PDFJSDev.test("TESTING")) {
+      return (
+        window.innerWidth *
+        window.innerHeight *
+        this.pixelRatio ** 2 *
+        (1 + capAreaFactor / 100)
+      );
+    }
+    return (
+      window.screen.availWidth *
+      window.screen.availHeight *
+      this.pixelRatio ** 2 *
+      (1 + capAreaFactor / 100)
+    );
```

Comment:
It just occurred to me that the pixelRatio can be a decimal number and that the capAreaFactor could be any integer (e.g. `123`), hence the resulting value could also become a decimal number which seems a bit strange.
While that probably won't cause any issues, it still seems nice to ensure that the result is always an integer.

```suggestion
      return Math.ceil(
        window.innerWidth *
        window.innerHeight *
        this.pixelRatio ** 2 *
        (1 + capAreaFactor / 100)
      );
    }
    return Math.ceil(
      window.screen.availWidth *
      window.screen.availHeight *
      this.pixelRatio ** 2 *
      (1 + capAreaFactor / 100)
    );
```